hackathon_site/event/serializers.py

```diff
@@ -34,6 +36,17 @@ class Meta:
         )
 
 
+class ProfileModifySerializer(serializers.Serializer):
```

Comment:
Use a `serializers.ModelSerializer`, then you don't need to specify the fields manually below. Just add their names to the `Meta.fields` tuple